crates/forge/tests/e2e/signing.rs

```diff
@@ -0,0 +1,26 @@
+use crate::assert_stdout_contains;
+use crate::e2e::common::runner::{runner, setup_package};
+use indoc::indoc;
+
+#[test]
+fn signing() {
+    let temp = setup_package("signing");
+    let snapbox = runner().arg("signing");
+
+    let output = snapbox.current_dir(&temp).assert().code(0);
+
+    assert_stdout_contains!(
+        output,
+        indoc! {r#"
+        [..]Updating[..]
+        [..]Compiling[..]
+        [..]Finished[..]
+
+
+        Collected 1 test(s) from signing package
+        Running 1 test(s) from src/
+        [PASS] signing::test
+        Tests: 1 passed, 0 failed, 0 skipped
+        "#}
+    );
+}
```

Comment:
I don't think we need an e2e test here 🤔 What does it verify above things we verify already in the integration?

Reply:
It gets oz account and verifies the signature generated by us. But we could probably get rid of that because it uses basically the same logic under the hood

Reply:
Oh ok, could we do this as integration test instead?

Reply:
Yup, but we would have to copy the OZ account 💀